lms/djangoapps/onboarding/decorators.py

```diff
@@ -12,15 +12,13 @@ def can_access_org_page(user_extended_profile):
     :param user_extended_profile:
     :return: boolean
     """
-    are_forms_complete = not (bool(user_extended_profile.unattended_surveys(_type='list')))
-
-    # user have completed profile & is admin of organization => can access org page
-    if are_forms_complete and user_extended_profile.organization and user_extended_profile.is_organization_admin:
+    # user is admin of organization => can access org page
+    if user_extended_profile.organization and user_extended_profile.is_organization_admin:
         can_access = True
 
     # user is at registration pages & signup as admin/first_learner => can access org page
-    elif not are_forms_complete and user_extended_profile.organization and \
-        (user_extended_profile.is_organization_admin or user_extended_profile.is_first_signup_in_org):
+    elif user_extended_profile.organization and (user_extended_profile.is_organization_admin
+                                                 or user_extended_profile.is_first_signup_in_org):
         can_access = True
     else:
         can_access = False
```

Comment:
all this can be changed to 
`
can_access = (user_extended_profile.organization and (user_extended_profile.is_organization_admin or user_extended_profile.is_first_signup_in_org))
`

Reply:
This PR is focused towards removing code. This decorator will be removed with the new regflow most probably
all this can also be changed to something even better than what you suggested:

```return bool(user_extended_profile.organization and (user_extended_profile.is_organization_admin or user_extended_profile.is_first_signup_in_org))```

Reply:
we probably dont need the bool there

Reply:
but I'm not optimizing code in this PR